### DISCUSSION TITLE
Default Group when opening Database

### DISCUSSION THREAD

**Discussion by KodeXall** [keepassxreboot/keepassxc#8431] [Q&A] [answered]
Is it possible to make KeepassXC to always show upon its opening a **_Group_** chosen by user. Or the _**Group**_ that was active when last closing took place?

If not- then I would like such option to be implemented

**Comment by phoerious** [comment, marked as answer]
KeePassXC will save which group was active last time you made any database changes.

**Reply by droidmonkey** [reply]
Correction, it will save the last selected group on LOCK. When you close the database or keepassxc that will be forgotten. That is on purpose.

**Reply by KodeXall** [reply]
I am guessing this is suppose to be one more tinny defense mechanism - but I reckon that if someone will be wit enough gain access to my file and get to know the password, then this will stop that person for about... what, a minute or two, before potentially wrecking a havoc to my life?

**Reply by phoerious** [reply]
The last used group is an ephemeral setting that doesn't get saved automatically, otherwise your database file would be rewritten constantly. The setting is therefore only saved when there are other changes too. It's not saved on lock as far as I remember, since that would also unexpectedly change the database.

**Reply by KodeXall** [reply]
OK, that sounds logical now - because there is no separate config file

But I see no reason for not having an ability to be able to set up the default **_Group_** - i.e. the most often used; or a dummy one with fake or less important logins / passwords, serving as a tiny defense element. Heck, some users might even have just one **_Group_**, so for them having to click it almost every single time must be extremely annoying

**Reply by droidmonkey** [reply]
If you only have one group... just store your entries in the root group.

**Reply by KodeXall** [reply]
Oh, snap...
